### PR TITLE
Hotfix sync tests

### PR DESCRIPTION
**Pull request by jcaballeros**

### CODE REVIEW

File: tests/check/gst/test_stream_sync.c [hunk 1]
```diff
@@ -73,38 +80,50 @@ GST_START_TEST (interpipe_synchronization)
...
   buffer_timestamp2 = GST_BUFFER_PTS (buffer);
   fail_if (buffer_timestamp2 == 0);
+
   fail_if (buffer_timestamp2 < buffer_timestamp1);
+  gst_sample_unref (outsample);
+
+  /* Now change to the first video src and pull another buffer */
+  g_object_set (G_OBJECT (intersrc), "listen-to", "intersink1", NULL);
+
+  outsample = gst_app_sink_pull_sample (GST_APP_SINK (asink));
```
Comment (mmontero) [inline tests/check/gst/test_stream_sync.c:121]: I am guessing this test verifies that interpipes can change between sources and keep a monotonically increasing pipeline, am I right? If so, how can you be sure this sample corresponds to the new source? you are not draining the pipeline before changing sources

File: tests/check/gst/test_stream_sync.c [hunk 2]
```diff
@@ -27,16 +29,21 @@
...
  * the other one, it should not be delay in the video when it is display
+ * only if the stream-sync property is set to compensate-ts (2)
  */
-GST_START_TEST (interpipe_synchronization)
+GST_START_TEST (interpipe_stream_sync_compensate_ts)
 {
   GstPipeline *sink1;
   GstPipeline *sink2;
+
+
```
[the remark this answers is not in the document]
Comment (jcaballeros) [reply tests/check/gst/test_stream_sync.c:41]: fixed

File: tests/check/gst/test_stream_sync.c [hunk 3]
```diff
@@ -27,16 +29,21 @@
...
   GstPipeline *src;
   GstElement *vtsrc1;
   GstElement *vtsrc2;
+
+
```
[the remark this answers is not in the document]
Comment (jcaballeros) [reply tests/check/gst/test_stream_sync.c:46]: fixed

File: tests/check/gst/test_stream_sync.c [hunk 4]
```diff
@@ -61,7 +68,7 @@ GST_START_TEST (interpipe_synchronization)
   /* Create one source pipeline */
   src =
       GST_PIPELINE (gst_parse_launch
-      ("interpipesrc name=intersrc listen-to=intersink1 block-switch=false allow-renegotiation=true format=3 ! capsfilter caps=video/x-raw,width=[320,1920],height=[240,1080],framerate=(fraction)5/1 ! "
+      ("interpipesrc name=intersrc listen-to=intersink1 stream-sync=compensate-ts block-switch=false allow-renegotiation=true format=3 ! capsfilter caps=video/x-raw,width=[320,1920],height=[240,1080],framerate=(fraction)5/1 ! "
```
[the remark this answers is not in the document]
Comment (jcaballeros) [reply tests/check/gst/test_stream_sync.c:71]: fixed

File: tests/check/gst/test_stream_sync.c (hunk 1 above)
Comment (jcaballeros) [reply tests/check/gst/test_stream_sync.c:130]: fixed

**Pull request merged**